src/pywbem/twisted_client.py

```diff
@@ -300,6 +302,109 @@ def parseResponse(self, xml):
         return [pywbem.tupleparse.parse_instance(x) for x in tt]
 
 
+class OpenEnumerateInstances(WBEMClientFactory):
+    """Factory to produce EnumerateInstances WBEM clients."""
+
+    def __init__(self, creds, classname, namespace='root/cimv2', **kwargs):
+        self.classname = classname
+        self.namespace = namespace
+        self.context = None
+
+        if not kwargs.get('MaxObjectCount'):
+            kwargs['MaxObjectCount'] = DEFAULT_ITER_MAXOBJECTCOUNT
+
+        payload = self.imethodcallPayload(
+            'OpenEnumerateInstances',
+            namespace,
+            ClassName=CIMClassName(classname),
+            **kwargs)
+
+        WBEMClientFactory.__init__(
+            self,
+            creds,
+            operation='MethodCall',
+            method='OpenEnumerateInstances',
+            object=namespace,
+            payload=payload)
+
+    def __repr__(self):
+        return '<%s(/%s:%s) at 0x%x>' % \
+               (self.__class__, self.namespace, self.classname, id(self))
+
+    def parseResponse(self, xml):
```

Comment:
Please use one naming style for the methods in the class.

Reply:
`parseResponse` it is an override of base parse method for WBEMClientFactory.

Reply:
In this case, you had to inherit the rule and apply it to other methods in the class, the get_rslt_params method doesn't fit the rule, use _getResultParams instead.